src/components/locationForm/components/TextInput.jsx

```diff
@@ -16,35 +16,25 @@ const StyledInput = styled(Input)`
     `}
 `;
 
-const TextInput = ({ name, label, defaultValue, onChange, ...otherProps }) => {
-  const [value, setValue] = useState('');
-
-  useEffect(() => {
-    setValue(defaultValue || '');
-  }, [defaultValue]);
-
-  const onValueChange = e => {
-    setValue(e.target.value);
-    onChange(e);
-  };
-
+const TextInput = ({ name, value, onChange, ...otherProps }) => {
   return (
     <StyledInput
       name={name}
-      value={value}
+      value={value || ''}
```

Comment:
The `value` prop already has a default value. Is this logic operator necessary?

Reply:
yes, this helps prevent assigning of the null values.